editor/src/clj/editor/outline_view.clj

```diff
@@ -458,9 +463,29 @@
       (ui/cancel future)
       (ui/user-data! cell :future-expand nil))))
 
+(defn- toggle-visibility! [node-outline-key-path ^MouseEvent event]
+  (ui/run-command (.getSource event) :hide-toggle {:node-outline-key-path node-outline-key-path})
+  (.consume event))
+
+(def eye-open-path (ui/load-svg-path "scene/images/eye_open.svg"))
+(def eye-closed-path (ui/load-svg-path "scene/images/eye_closed.svg"))
+
 (defn make-tree-cell
   [^TreeView tree-view drag-entered-handler drag-exited-handler]
-  (let [cell (proxy [TreeCell] []
+  (let [eye-icon-open (app-view/make-svg-icon-graphic eye-open-path)
```

Comment:
Not that important, but should we move `make-svg-icon-graphic` under `editor.ui.clj`, or maybe `editor.icons.clj`?